catalog-info.yaml

```diff
@@ -0,0 +1,12 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-components-tinymce-advanced-plugins'
+  description: "the standard edX header for use in React applications"
```

Comment:
This description seems wrong, it looks like the README has a typo.  I think you can say:

"A set of tinymce plugins to use in react applications."